R/model-wrappers.R

```diff
@@ -290,3 +290,68 @@ forecastHybrid_model <- function(y = NULL, samples = NULL,
   return(sample_from_model)
 
 }
+
+
+
+
+#' forecast model wrapper
+#'
+#' Allows users to forecast using models from the `forecast`
+#' Note that `forecast` must be installed for this model wrapper to be functional.
+#' @param model A `forecast` model object.
+#' @inheritParams bsts_model
+#' @export
+#' @return A dataframe of predictions (with columns representing the
+#' time horizon and rows representing samples).
+#'
+#' @importFrom stats ts
+#' @importFrom purrr map
+#' @importFrom dplyr bind_rows
+#'
+#' @examples \dontrun{
+#'
+#' ## Used on its own
+#' forecast_model(y = EpiSoon::example_obs_rts[1:10, ]$rt,
+#'                model = forecast::auto.arima,
+#'                samples = 10, horizon = 7)
+#'
+#'
+#' forecast_rt(EpiSoon::example_obs_rts[1:10, ],
+#'             model = function(...){
+#'             forecast_model(model = forecast::ets, ...)},
+#'             horizon = 7, samples = 10)
+#'}
+#'
+
+forecast_model <- function(y = NULL, samples = NULL,
```

Comment:
no argument passing

Reply:
sorry still unsure what you mean

Reply:
if I added (...) (which we might want to have e.g. for external regressors) I will run in the same problem I had with the last wrapper where arguments get passed to the wrong level. So just copied the way fable_model handles it

Reply:
huh. maybe it does work. Will update and you can see whether it is what you intended

Reply:
have you tested passing arguments into `forecast_model` from `forecast_rt` for the individual models - that is what needs to work

Reply:
that works now :)